modules/op-resolver-git-scm.adoc

```diff
@@ -68,25 +66,28 @@ The following example pipeline run references a remote pipeline from a Git repos
 
 [source,yaml]
 ----
-apiVersion: tekton.dev/v1
+apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
   name: git-pipeline-reference-demo
 spec:
   pipelineRef:
     resolver: git
     params:
-    - name: url
-      value: https://github.com/tektoncd/catalog.git
-    - name: revision
-      value: main
-    - name: pathInRepo
-      value: pipeline/simple/0.1/simple.yaml
-    - name: sample-pipeline-parameter
-      value: test
+      - name: repo
+        value: catalog
+      - name: org
+        value: tektoncd
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipeline/simple/0.1/simple.yaml
+      - name: sample-pipeline-parameter
+        value: test
```

Comment:
Same here, why are we putting this "random" parameter here ? It might render the request invalid.